entity/src/main/java/dev/ikm/tinkar/entity/graph/adaptor/axiom/LogicalExpressionBuilder.java

```diff
@@ -41,6 +42,10 @@ public class LogicalExpressionBuilder {
     DiTreeEntity.Builder builder;
     final int rootIndex;
 
+    static UUID generateRandomUuid() {
+        return UuidT5Generator.get(UuidT5Generator.STAMP_NAMESPACE, String.valueOf(System.nanoTime()));
```

Comment:
What are the odds of two threads calling this at the exact same time? The danger I see is if this happens with a constant namespace, they would have the same uuid, whereas if we pass in a namespace generated based on the thread's name, it would be unique.

Reply:
> What are the odds of two threads calling this at the exact same time? The danger I see is if this happens with a constant namespace, they would have the same uuid, whereas if we pass in a namespace generated based on the thread's name, it would be unique.

Updated to prepend `Thread.currentThread().getName()`